sdk/tests/conformance2/buffers/get-buffer-sub-data.html

```diff
@@ -168,6 +168,20 @@
 gl.deleteBuffer(buffer);
 shouldBeTrue("areArraysEqual(retArray, floatArray)");
 
+debug("");
+debug("Test that getBufferSubData...");
+var transformFeedbackBuffer = gl.createBuffer();
+gl.bindBuffer(gl.TRANSFORM_FEEDBACK_BUFFER, transformFeedbackBuffer);
+gl.bufferData(gl.TRANSFORM_FEEDBACK_BUFFER, floatArray, gl.STATIC_DRAW);
+var transformFeedback = gl.createTransformFeedback();
+debug("... passes when a transform feedback object is not bound");
+wtu.shouldGenerateGLError(gl, gl.NO_ERROR, "gl.getBufferSubData(gl.TRANSFORM_FEEDBACK_BUFFER, 0, retArray, retArray.length)");
+gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, transformFeedback);
```

Comment:
It should be beginTransformFeedback() that causes dual usage of the buffer with getBufferSubData. Per our offline discussion, it's easier to add this to the transform_feedback.html test.